Source/ASDisplayNode.mm

```diff
@@ -3271,6 +3271,18 @@ - (UIEdgeInsets)hitTestSlop
   return _hitTestSlop;
 }
 
+- (BOOL)isRTL {
+  if (AS_AVAILABLE_IOS_TVOS(10, 10)) {
+    return _primitiveTraitCollection.layoutDirection == UITraitEnvironmentLayoutDirectionRightToLeft;
```

Comment:
You need to either acquire the instance lock before accessing this ivar or use the setter.

```suggestion
    return self.primitiveTraitCollection.layoutDirection == UITraitEnvironmentLayoutDirectionRightToLeft;
```